packages/core/useFetch/index.ts

```diff
@@ -90,6 +90,7 @@ export interface UseFetchReturn<T> {
 
 type DataType = 'text' | 'json' | 'blob' | 'arrayBuffer' | 'formData'
 type HttpMethod = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH' | 'HEAD' | 'OPTIONS'
+type Combination = 'overwrite' | 'combineCallbacksin'
```

Comment:
I'm wondering if `chain` would not be a better word here than `combineCallbacksin`

Reply:
I means to be combine. It turns out pushed a strange word.